pandas/core/tools/datetimes.py

```diff
@@ -976,7 +933,7 @@ def to_datetime(
     in addition to forcing non-dates (or non-parseable dates) to :const:`NaT`.
 
     >>> pd.to_datetime('13000101', format='%Y%m%d', errors='ignore')
-    datetime.datetime(1300, 1, 1, 0, 0)
+    '13000101'
```

Comment:
this one also looks like it was wrong to begin with - we don't return `datetime.datetime` with `errors='ignore'` for any other format, e.g.
```
In [62]: to_datetime('1300', format='%Y', errors='ignore')
Out[62]: '1300'
```

Reply:
are there many other cases where we get pydatetime objects instead of Timestamps?  If no, then i think it makes sense to tighten and say "you always get Timestamps or ignore/raise".  If yes, then i think returning the pydatetime objects is better here than the string (and probably also for '1300' with '%Y')